c2corg_api/views/document_changes.py

```diff
@@ -0,0 +1,141 @@
+from c2corg_api.models import DBSession
+from c2corg_api.models.document import Document, ArchiveDocumentLocale
+from c2corg_api.models.document_history import DocumentVersion, HistoryMetaData
+from c2corg_api.models.user import User
+from c2corg_api.views import cors_policy
+from c2corg_api.views.feed import get_params
+from c2corg_api.views.validation import validate_simple_token_pagination,\
+  validate_user_id_not_required
+
+from sqlalchemy.sql.expression import desc
+
+from cornice.resource import resource, view
+from sqlalchemy.orm import joinedload, load_only
+
+
+@resource(path='/documents/changes', cors_policy=cors_policy)
+class ChangesDocumentRest(object):
+    """ Unique class for returning history of changes in documents.
+    """
+
+    def __init__(self, request):
+        self.request = request
+
+    @view(validators=[validate_simple_token_pagination,
+                      validate_user_id_not_required])
+    def get(self):
+        """Get the public document changes feed.
+
+        Request:
+            `GET` `/document/changes[&limit=...][&token=...][&u=]`
+
+        Parameters:
+
+            `limit=...` (optional)
+            How many entries should be returned (default: 30).
+            The maximum is 50.
+
+            `token=...` (optional)
+            The pagination token. When requesting a feed, the response includes
+            a `pagination_token`. This token is to be used to request the next
+            page.
+
+            `u=...` (optional)
+            Changes made by one user.
+
+            For more information about "continuation token pagination", see:
+            http://www.servicedenuages.fr/pagination-continuation-token (fr)
+        """
+        user_id = self.request.validated.get('u')
+        lang, token_id, token_time, limit = get_params(self.request, 30)
```

Comment:
Minor: You could write `lang, token_id, _, limit = ...` to indicate that you are not interested in the value of `token_time`. This is a common pattern in Haskell, but you also find it in Python code.